### PR TITLE
fix: overlapping sections on landing page

### PR DESCRIPTION
**Pull request by JStoresund**
Solution: changed some styling in the companyhero component; the changes are inspired by the companycard component styling.

### CODE REVIEW

**Review by AasmundN — COMMENTED** [review]
The bottom of the company hero text was still cut off. Think I fixed it. Someone else will have to review.

File: components/company/Hero.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 <template>
   <div>
-    <VSheet :width="xs ? '90vw' : '65vw'">
+    <VCard :width="xs ? '90vw' : '65vw'" md="50v" variant="flat">
```
Comment (ipeglin) [inline components/company/Hero.vue:29]: I think we should set a max height for every viewport. When the height is the same as this defined limit, for any given viewport, the description should be truncated with `nowrap` and `ellipsis`

**Review by ipeglin — CHANGES_REQUESTED** [review]
I think it looks really good. But I think we can increase the logo size a bit, and truncate the description when companyName + description exceeds the height of the logo

**Review by AasmundN — APPROVED** [review]
Nice! Looks better. But if the text overflows, where is the user supposed to read the rest of it? Maybe on a separate page, idk.

**Pull request merged**